### PR TITLE
Games

### PR DESCRIPTION
**Pull request by anthonyme00**
-Minesweeper game!
commands:
<prefix>minesweeper start|end
self explanatory (you can add difficulty modifier after start (currently easy,medium,hard, or preset))
<prefix>reveal <number> <character>
reveal a tile
<prefix>flag <number> <character>
flag a tile to disallow revealing that tile
<prefix>unflag <number> <character>
unflag a tile

Also a better ascii command

### CODE REVIEW

File: src/commands/fun/asciify.ts [hunk 1]
```diff
@@ -47,12 +47,12 @@ export default class extends Command<BotClient> {
         };
 
         return request(options).then(asciified => {
-            const embed = new RichEmbed();
-            embed.setTitle('ASCIIFY');
-            embed.setURL(options.url);
-            embed.setAuthor(message.author.username, message.author.avatarURL);
-            embed.addField('ASCII ART', this.formatAscii(asciified), false);
-            message.channel.send({ embed });
+            // const embed = new RichEmbed();
```
Comment (ttbowen) [inline src/commands/fun/asciify.ts]: Could we remove these comments now?

File: src/commands/games/minesweeper.ts [hunk 2]
```diff
@@ -0,0 +1,399 @@
...
+                );
+                const lossModifier = lose[Math.round(Math.random() * (lose.length - 1))];
+                return message.channel.sendMessage('Game finished... You lost. ' + lossModifier);
+            }
+        }
+
+        return message.channel.send(this.constructGameScreen(game));
+    }
+
+    async flag(message: Message, args: any[]): Promise<any> {
```
Comment (ttbowen) [inline src/commands/games/minesweeper.ts]: Make this method `private`. Only `action` should be `public` on `Command` classes.

File: src/commands/fun/asciify.ts (hunk 1 above)
Comment (anthonyme00) [reply src/commands/fun/asciify.ts]: Definitely. I just forgot to remove those

Comment (ttbowen) [reply src/commands/fun/asciify.ts]: Oh and is `formatAscii` now redundant? If so, I would remove that too.

**Pull request merged**